app/src/main/java/ch/sdp/vibester/helper/GameManager.kt

```diff
@@ -78,7 +81,11 @@ open class GameManager : Serializable {
      */
     @JvmName("getCurrentSong1")
     fun getCurrentSong(): Song {
-        return currentSong
+        if(this::currentSong.isInitialized) {
+            return currentSong
+        } else {
+            return Song.songBuilder("http://example.com", "http://example.com", songName, artistName)
```

Comment:
what is the purpose if this song.builder?

Reply:
The purpose of this song builder is that when the currentSong is not initialized yet (mean no web connection for instance/low connection), it does not crash and has song title and name

Reply:
I am not sure whether we should put a fake song. maybe we can put some text to screen. lets fix it next pr!